lib/modules/manager/tekton/extract.ts

```diff
@@ -69,7 +69,7 @@ function addDep(ref: TektonBundle, deps: PackageDependency[]): void {
   let imageRef: string | undefined;
   // Find a bundle reference from the Bundle resolver
   if (ref.resolver === 'bundles') {
-    for (const field of coerceArray(ref.resource)) {
+    for (const field of coerceArray(ref.params)) {
```

Comment:
we should support the deprecated resources and provide a migration instead.

a lot of our managers support deprecated features.

Reply:
ack, makes sense. done